paypal/express/gateway.py

```diff
@@ -25,12 +25,15 @@
 
 SALE, AUTHORIZATION, ORDER = 'Sale', 'Authorization', 'Order'
 
+
 # The latest version of the PayPal Express API can be found here:
 # https://developer.paypal.com/docs/classic/release-notes/
 API_VERSION = getattr(settings, 'PAYPAL_API_VERSION', '119')
 
 logger = logging.getLogger('paypal.express')
 
+buyer_pays_on_paypal = lambda: getattr(settings, 'PAYPAL_BUYER_PAYS_ON_PAYPAL', False)
```

Comment:
Tests: if it's a constant I can't use `override_settings`.
https://code.djangoproject.com/ticket/22071